packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -291,6 +292,10 @@ class CheckboxListTile extends StatelessWidget {
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
   final bool? enableFeedback;
 
+  /// if enabled = false, the [CheckboxListTile] will be disabled and will not react to user input.
```

Comment:
This doesn't really match the standard format for Flutter class comments. See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements

This feature is supposed to work like the ListTile version, so you could use that as a model: https://api.flutter.dev/flutter/material/ListTile/enabled.html

Reply:
done